qemu/tests/cfg/live_snapshot.cfg

```diff
@@ -73,6 +69,11 @@
                 - stop_cont:
                     before_start = "stop"
                     after_finished = "resume reboot verify_alive"
+                - reboot:
```

Comment:
it's better to update the name, otherwise it will be run when we run case "reboot"